<?xml version="1.0" encoding="ISO-8859-15"?>

<launch>

    <arg name="image_topic" default="/rgb/image_raw"/>
    <arg name="image_encoding" default="bgr8" />
    <arg name="dataset_path" default="$(find kimera_interface)/records/" />
    <arg name="throttle_msgs" default="5" />

    <arg name="sensor_frame" default="hp_laptop_link" />
    <arg name="parent_frame" default="hp_laptop_odometry_frame" />
    

    <node name="image_pose_recorder_node" pkg="kimera_interface" type="save_images_with_poses.py">
       
        <param name="image_topic" value="$(arg image_topic)"/>
        <param name="image_encoding" value="$(arg image_encoding)" />
        <param name="dataset_path" value="$(arg dataset_path)" />
        <param name="throttle_msgs" value="$(arg throttle_msgs)" />

        <param name="sensor_frame" value="$(arg sensor_frame)" />
        <param name="parent_frame" value="$(arg parent_frame)" />
        
    </node>

</launch>
